tests/ISA/ISA.Tests/ISA.Tests.fsproj

```diff
@@ -23,12 +24,18 @@
     <Compile Include="Fable.Tests.fs" />
     <Compile Include="Main.fs" />
   </ItemGroup>
+  <ItemGroup />
   <ItemGroup>
     <!--Also includes ISA-->
     <ProjectReference Include="..\..\..\src\ISA\ISA\ARCtrl.ISA.fsproj" />
-    <ProjectReference Include="..\..\TestingUtils\TestingUtils.fsproj" />
```

Comment:
Necessary? Can you explain the problem again?

Reply:
We talked about this, TestingUtils contains references to ARC main project, which makes testing for python way more difficult

Reply:
Ah I see, so this would be a temporary thing? Alternatively we could split TestObjects and TestUtils. But I'll open up an issue.